donkeycar/pipeline/training.py

```diff
@@ -153,7 +153,7 @@ def train(cfg: Config, tub_paths: str, model: str = None,
     assert val_size > 0, "Not enough validation data, decrease the batch " \
                          "size or add more data."
     logger.info(f'Train with image caching: '
-                f'{getattr(cfg, "CACHE_IMAGES", True)}')
+                f'{getattr(cfg, "CACHE_IMAGES", "ARRAY")}')
```

Comment:
@DocGarbanzo I don't know that this means.  Does this limit the size of the cache?

Reply:
In the previous PR, we have upgraded the CACHE_IMAGES config parameter to allow for 3 values, i.e NOCACHE, ARRAY and BINARY, please see here: https://github.com/autorope/donkeycar/pull/1173/files#diff-8a130617f883f549bf089e3ab0c46cabd4c3e4a69b50568c3b27ef73414e7c28. ARRAY represents the original setting where the whole image is cached as a `np.uint8` array (consuming about 58k of mem ~ 120 * 160 * 3). With BINARY we can now cache the image as binary jpeg encoded which uses only 4k of memory. This caching still provides significantly faster access to the image than re-reading it from disk, even with a fast M2 SSD. With that much smaller memory footprint we can now train >> 100k images with caching, speeding up the training on large datasets by a lot. In this line here, I just fixed a left-over from the previous PR and here, it only affects logging the caching status into the shell, because the default changed from the binary `True` to the enum/string `'ARRAY'`.